tokenutil/token_util.go

```diff
@@ -47,25 +47,29 @@ func GetRegistryRefreshToken(registry, resourceID, clientID string) (string, err
 
 	req, err := http.NewRequest("POST", exchangeURL, strings.NewReader(v.Encode()))
 	if err != nil {
-		return "", errors.Wrap(err, "unable to create an HTTP post request to exchangeAPI")
+		return "", errors.Wrap(err, "unable to create the request to get ACR refresh token")
 	}
 	req.Header.Set("Content-Type", "application/x-www-form-urlencoded")
 
 	response, err := client.Do(req)
 	if err != nil {
-		return "", errors.Wrap(err, "unable to make post request to exchangeAPI")
+		return "", errors.Wrap(err, "unable to make the request to get ACR refresh token")
```

Comment:
unable to make sounds a bit odd. Should we do "unable to send..." instead, it seems more clear